docs/tutorials/full_finetuning.md

```diff
@@ -54,7 +54,7 @@ The conversion scripts do not use accelerators but need large host memory to per
 - For large size model (e.g. 70B model), this script requires large memory VM.
 - The script load and save weights in a single pass.
 
-### Sample Full Fine tuning Script
+### Sample full fine tuning script
```

Comment:
Should we consistently say either "fine tuning" or "finetuning"? Both appear in the documentation in different documents.